src/vs/workbench/parts/terminal/electron-browser/terminalActions.ts

```diff
@@ -227,6 +228,64 @@ export class MoveToLineEndTerminalAction extends BaseSendTextTerminalAction {
 	}
 }
 
+export class SendSequenceTerminalCommand extends Command {
```

Comment:
Does it need to be a Command not an Action to support args?

Reply:
I tried to look for that. I could not find any way to implement this as Action. If there exist some way, I would not mind making changes accordingly.